test/integration/BUILD

```diff
@@ -1216,6 +1216,30 @@ envoy_cc_test(
     ],
 )
 
+envoy_proto_library(
+    name = "starttls_integration_proto",
+    srcs = [":starttls_integration_test.proto"],
+)
+
+envoy_cc_test(
+    name = "starttls_integration_test",
+    srcs = [
+        "starttls_integration_test.cc",
+    ],
+    data = [
+        "//test/config/integration/certs",
+    ],
+    deps = [
+        ":integration_lib",
+        ":starttls_integration_proto_cc_proto",
+        "//source/extensions/filters/network/tcp_proxy:config",
+        "//source/extensions/transport_sockets/raw_buffer:config",
+        "//source/extensions/transport_sockets/starttls:config",
```

Comment:
We introduced a new direct dependency between core tests and //source/extensions.
Could we move this test to //test/extensions/transport_sockets ?

Reply:
Oops sorry I missed this. Yes @cpakulski please move.

Reply:
No problem - will move it.

Reply:
Thanks!

Reply:
Moved the test: https://github.com/envoyproxy/envoy/pull/14425